.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: pg-unit-testing
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  pgaux:
+    runs-on: ubuntu-latest
+    container: perl:latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: run a simple unit test
+        run: |
+          cpanm Data::Dump
+          perl t/macros/pgaux.t
```

Comment:
generally it's nicer to use `prove` to run the test rather than just `perl`

Reply:
I see, I'll change this line in the yaml file, then.

Reply:
I just pulled this in. I'll make the change.